autotelic/fastify-stream-to-csv
# feat: types

# Summary

- Add type declarations so fastify-stream-to-csv can be used in typescript projects
- Update linting to use `@autotelic/eslint-config-js`
- Update dependencies (resolve dependabot)

# Test Plan

- Run tests
- Confirm plugin is compatible with a ts consumer - This PR is configured with this branch: https://github.com/autotelic/shakti-v2/pull/19
